src/Service/CognitoIdentityProvider/src/Input/SetUserMFAPreferenceRequest.php

```diff
@@ -16,14 +16,14 @@ final class SetUserMFAPreferenceRequest extends Input
      *
      * @var SMSMfaSettingsType|null
      */
-    private $SMSMfaSettings;
+    private $sMSMfaSettings;
```

Comment:
Properties like these are not super pretty.

Reply:
You're right. I've submitted a patch to fix that